README.md

```diff
@@ -28,6 +28,9 @@ More complete install documentation is available on the PostgreSQL [wiki](https:
 
 **Note:** If you're setting up the project on Mac OSX, we recommend using [homebrew](http://brew.sh) to install PostgreSQL.
 
+### Install Memcached
```

Comment:
Sure, I can move the new requirements to a new file (something like requirements-caching.txt) I'll go ahead and do that